### PR TITLE
update gramine ratls lib and initialise

### PR DESCRIPTION
**Pull request by lonerapier**

### CODE REVIEW

**Comment by lead4good** [comment]
lgtm

**Comment by lead4good** [comment]
@metachris Please review

File: main.go [hunk 1]
```diff
@@ -82,6 +84,10 @@ func main() {
 	srv, err := server.NewServer(serverOpts)
 	perr(err)
 
+	// Initialise RATLS library
+	err = ratls.InitRATLSLib(true, time.Hour, false)
```
Comment (metachris) [inline main.go:88]: i think this should only be included in a build with the sgx tag?

File: server/node_nosgx.go [hunk 2]
```diff
@@ -10,6 +10,11 @@ import (
 	"go.uber.org/zap"
 )
 
+// InitializeRATLSLib is a no-op for non-SGX builds
+func InitializeRATLSLib(_ bool, _ time.Duration, _ bool) error {
```
Comment (metachris) [inline server/node_nosgx.go:14]: I don't like this way, it doesn't feel clean.

For this I think I'd prefer an init function on `node_sgx.go` that panics on error, and no changes to the regular node or server.

Comment (lonerapier) [reply server/node_nosgx.go:14]: @metachris what if in future user wants to change ratls lib config like `timeout duration`, an `init` func wouldn't allow that right? My intention of adding it this way was that it allows to easily change config and maybe take argument through command line.

**Comment by metachris** [comment]
lgtm, thx!

**Pull request merged**